package.json

```diff
@@ -36,25 +44,27 @@
     "fuse.js": "^3.1.0",
     "immutable-props": "^1.1.0",
     "isdev": "^1.0.1",
-    "knex": "^0.13.0",
+    "knex": "^0.14.0",
```

Comment:
is this version change necessary? It might not be compatible with other dep versions we have